src/test/java/seedu/address/logic/parser/DeleteCommandParserTest.java

```diff
@@ -29,4 +29,9 @@ public void parse_validArgs_returnsDeleteCommand() {
     public void parse_invalidArgs_throwsParseException() {
         assertParseFailure(parser, "a", MESSAGE_INVALID_PERSON_DISPLAYED_INDEX);
     }
+
+    @Test
+    public void parse_noInfex_throwsParseException() {
```

Comment:
i believe there is a spelling error here?

Reply:
Yes sir thank you for pointing that out